Compilation error in Alpine

Getting the following errors
(the `ncurses-dev` package for Alpine is installed)

```
src/Writer/NcursesWriter.cpp: In member function 'void vis::NcursesWriter::write_background(int32_t, int32_t, vis::ColorIndex, const wstring&)':
src/Writer/NcursesWriter.cpp:62:41: error: 'mvaddwstr' was not declared in this scope
     mvaddwstr(height, width, msg.c_str());
                                         ^
src/Writer/NcursesWriter.cpp: In member function 'void vis::NcursesWriter::write_foreground(int32_t, int32_t, vis::ColorIndex, const wstring&)':
src/Writer/NcursesWriter.cpp:73:41: error: 'mvaddwstr' was not declared in this scope
     mvaddwstr(height, width, msg.c_str());
                                         ^
make: *** [Makefile:233: /cli-visualizer/build/NcursesWriter.o] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -42,6 +42,7 @@ CXX_FLAGS += -march=$(VIS_COMPILER_ARCH)
 CXX_FLAGS += -ffast-math
 CXX_FLAGS += -fno-omit-frame-pointer
 CXX_FLAGS += -D__extern_always_inline=inline
+CXX_FLAGS += -D_XOPEN_SOURCE_EXTENDED
 
 TEST_CCACHE_CLANG=ccache clang++
 TEST_CLANG=clang++
@@ -68,7 +69,7 @@ PERF_TEST_CXX_FLAGS += -ggdb -g2
 
 # Mac OS
 ifeq ($(OS),Darwin)
-CXX_FLAGS += -dynamic -D_OS_OSX -D_XOPEN_SOURCE_EXTENDED
+CXX_FLAGS += -dynamic -D_OS_OSX
 
 # Linux
 else

